src/plugins/navigation/next.js

```diff
@@ -163,7 +163,14 @@ export default pluginFactory({
                     unansweredOnly: unansweredOnly
                 });
 
-                if (warningHelper.shouldWarnBeforeEnd()) {
+                if (warningHelper.shouldWarnBeforeEndPart()) {
+                    testRunner.trigger(
+                        'confirm.endTestPart',
+                        __('You are about to go to the next item. Click OK to continue and go to the next item.'),
```

Comment:
I don't think this is the correct message. From ticket I see 
```
You are about to submit this test part.

There are X unanswered questions in this part of the test
```

Reply:
I just push my changes for allow others to pick my task, but I will do :)